src/pages/ContentScripts/features/repo-header-labels/view.tsx

```diff
@@ -37,6 +37,27 @@ const View = ({ activity, openrank, participant, contributor, meta }: Props): JS
   const participantData = generateDataByMonth(participant, meta.updatedAt);
   const contributorData = generateDataByMonth(contributor, meta.updatedAt);
 
+  type Label = {
+    id: string;
+    name: string;
+    type: string;
+  };
+
+  const labels: Label[] = meta.labels as Label[];
+
+  const spans = labels.map((label, index) => (
```

Comment:
The second parameter `index` is not used in the subsequent code and can be deleted.